test-harness/modules/mocker/endpoints/marathon.py

```diff
@@ -352,7 +352,10 @@ def _calculate_response(self, base_path, *_):
 
         with ctx.lock:
             if base_path == '/v2/apps':
-                blob = self._convert_data_to_blob(ctx.data['endpoint-content'])
+                if ctx.data["endpoint-content-raw"]:
```

Comment:
`endpoint-content-is-raw` ?

Remember that Marathon endpoint and others will only keep getting more complex as we add tests. Lets make our lifes easier by naming stuff in easy to understand manner :)